src/styles/accessibility.css

```
@@ -0,0 +1,127 @@
+  }
+}
+
+/* Color blind friendly indicators */
+.active[role="tab"]::before {
+  content: '✓ ';
+  font-weight: bold;
+}
+
+/* Better error message accessibility */
+.error-message {
+  role: alert;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)